decidim-proposals/app/views/decidim/proposals/admin/proposals/index.html.erb

```diff
@@ -6,7 +6,9 @@
         <%= t(".title") %>
         <span id="js-selected-proposals-count" class="component-counter component-counter--inline" title="<%= t("decidim.proposals.admin.proposals.index.selected") %>"></span>
       </div>
-      <%= render partial: "bulk-actions" %>
+
```

Comment:
Just for consistency, although I could be on board of making some distinction/separation between the actions and just informational/metadata, but we should do it in all the rest of the components 😄 
```suggestion
```